torch/ao/sparsity/experimental/pruner/base_pruner.py

```diff
@@ -53,43 +53,14 @@ class BasePruner(abc.ABC):
             be updated.
 
     """
-    def __init__(self, model, config, defaults):
+    def __init__(self, defaults):
```

Comment:
Is it possible to make the BasePruner inherit from the Base Sparsifier? This way code for setstate/getstate etc can be shared.